### ISSUE TITLE
Skype-Status invisible of the contacts

### ISSUE THREAD

**Issue by estrellita80** [EionRobb/skype4pidgin#356]
Before July, status of the contacts in invisible mode was shown as "Invisible". But now, this status isn't appear, only "offline". 
Is it a bug? is it possible show "invisible" again?

**Comment by EionRobb** [comment]
Unfortunately that would be something out of control of the plugin. It's up
to the server to pass on the Invisible status and was probably a bug all
along that invisible buddies could be seen.

**Comment by estrellita80** [comment]
First, thank you by your answer.

I discovered something about the invisible status. It may not be of
interest, but I commeted it.

When an user is offline (disconnected) and turn on skype in invisible mode,
the register (pidgin) shows "the status of "user" changed from offline to
offline". I checked it and the status is invisible.

But when the user disconnect after there is not register.

I think that is posssible created an alarm of this event.

It's just a suggestion.

Thank you!

2015-11-01 22:19 GMT+01:00 Eion Robb <notifications@github.com>:
